src/strategies/IchiQuickSwapMerklFarmStrategy.sol

```diff
@@ -162,8 +162,12 @@ contract IchiQuickSwapMerklFarmStrategy is LPStrategyBase, FarmingStrategyBase {
     /// @inheritdoc IStrategy
     function getSpecificName() external view override returns (string memory, bool) {
         IFactory.Farm memory farm = _getFarm();
-        string memory shortAddr = IQMFLib.shortAddress(farm.addresses[0]);
-        return (shortAddr, true);
+        IICHIVault _ivault = IICHIVault(farm.addresses[0]);
+        address allowedToken;
```

Comment:
write something like
```solidity
address allowedToken = _ivault.allowToken0() ? _ivault.token0() : _ivault.token1();
```
to save gas and reduce branches

Reply:
thank you. I fixed issues.